docs/rules/no-import-assign.md

```diff
@@ -18,6 +18,7 @@ mod = 1          // ERROR: 'mod' is readonly.
 named = 2        // ERROR: 'named' is readonly.
 mod_ns.named = 3 // ERROR: the members of 'mod_ns' is readonly.
 mod_ns = {}      // ERROR: 'mod_ns' is readonly.
+Object.assign(mod_ns, { foo: "foo" }) // ERROR: 'mod_ns' is not extensible.
```

Comment:
```suggestion
Object.assign(mod_ns, { foo: "foo" }) // ERROR: The members of 'mod_ns' are readonly.
```

Reply:
As shown in [Online Demo](https://eslint.org/demo#eyJ0ZXh0IjoiLyogZXNsaW50IG5vLWltcG9ydC1hc3NpZ246IFwiZXJyb3JcIiAqL1xuXG5pbXBvcnQgKiBhcyBtb2RfbnMgZnJvbSBcIi4vbW9kLm1qc1wiXG5cbk9iamVjdC5hc3NpZ24obW9kX25zLCB7IGZvbzogXCJmb29cIiB9KTsiLCJvcHRpb25zIjp7InBhcnNlck9wdGlvbnMiOnsiZWNtYVZlcnNpb24iOjEyLCJzb3VyY2VUeXBlIjoibW9kdWxlIiwiZWNtYUZlYXR1cmVzIjp7fX0sInJ1bGVzIjp7fSwiZW52Ijp7fX19)

Reply:
but in this case, it was to extend `mod_ns`, not to write the members.

Reply:
Extending is the same as writing new members. I think we should not change the Error message here as it is coming from ESLint, tough we can add a comment above to show it's not extensible:

```suggestion
// Can't extend 'mod_ns'
Object.assign(mod_ns, { foo: "foo" }) // ERROR: The members of 'mod_ns' are readonly.
```

Reply:
makes sense! 👍